### PR TITLE
[GAPRINDASHVILI] Allow mapping Target Datasources that have less allocated space than Source Datastores

### PR DESCRIPTION
**Pull request by AparnaKarve**
Upstream PR https://github.com/ManageIQ/miq_v2v_ui_plugin/pull/408

BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1595446

### CODE REVIEW

File: app/javascript/react/screens/App/Overview/screens/MappingWizard/components/MappingWizardDatastoresStep/components/DatastoresStepForm/helpers.js [hunk 1]
```diff
@@ -22,34 +20,16 @@ export const targetDatastoreAvailableSpace = (targetDatastore, datastoresStepMap
...
-    __('%s (%s total, %s avail)'),
-    targetDatastore.name,
-    numeral(total_space).format('0.00b'),
-    numeral(availableSpace).format('0.00b')
-  );
-};
+export const targetDatastoreTreeViewInfo = targetDatastore => targetDatastore.name;
 
-export const errorMessage = __('The size of the selected source datastores exceeds the available space in the target datastore'); // prettier-ignore
+export const errorMessage = __(
```
Comment (AllenBW) [inline app/javascript/react/screens/App/Overview/screens/MappingWizard/components/MappingWizardDatastoresStep/components/DatastoresStepForm/helpers.js]: waitttt i think we still want this to all be on one line... the `__(` n stuff

Comment (AparnaKarve) [reply app/javascript/react/screens/App/Overview/screens/MappingWizard/components/MappingWizardDatastoresStep/components/DatastoresStepForm/helpers.js]: Good catch - thanks!

**Comment by AllenBW** [comment]
@AparnaKarve Any chance we can delete this branch from the v2v repo and yah recreate the pr on yer own fork?

**Comment by AparnaKarve** [comment]
Closing this in favor of https://github.com/ManageIQ/miq_v2v_ui_plugin/pull/492

**Pull request closed without merging**